projects.md

```diff
@@ -0,0 +1,70 @@
+# Projects
+
+## Sales Prospect & Internal Proposals
+
+While the Business Planning WG is mainly in charge of sales lead generation,
+all members may pursue leads as they come up.
+These may be a client or a grant opportunity.
+Members are strongly encouraged to record leads into the co-operative's [leads tracking system](https://link.hypha.coop/leads) whether or not we want to pursue this lead.
+
+If a member decides to pursue a **lead**,
+they can reach out to them to see whether there's potential to work together.
+If the lead expresses intention to proceed with a concrete project in mind,
+this becomes a **prospect**.
+The member now heads up this sales effort and should draft an **internal proposal** and present it at a meeting to see whether the co-operative wants to move forward to submit a formal proposal to the prospective client.
+In this meeting, we should discuss any concerns by other members,
+and gauge team capacity given project timelines.
+A sample internal proposal should include:
+
+- Project Title
+- Background
+- Scope & Timeline
+- Outcomes & Opportunities
+- Skillsets
+- Budget
+- Communication
+- Risks
+- Resources
+
+A [template](#internal-proposal-template) is included below. You can add more sections as appropriate for the project. This may include sample code, project roadmap, next steps, etc. You can find examples in our archived proposals folder in our [shared drive](http://link.hypha.coop/drive).
+
+## Client Proposal Drafting and Submissions
+
+Project proposals to be submitted to a potential client or grant committee should be drafted by members who expect to participate in the project with the help of members of the Business Planning WG. The WG has expertise in drafting client and grant project proposals, and managing the proposal process, so it is helpful to involve members of that group from the drafting stage. All that are involved in the proposal drafting make up the **proposal team** that will eventually sign off on the final draft to be submitted.
```

Comment:
```suggestion
Project proposals that will be submitted to a potential client or to a grant committee should be drafted by members who expect to participate in the project with the help of Business Planning WG. Members of the WG have expertise drafting project proposals and managing the proposal process, so it is helpful to involve them from the drafting stage. All those involved in the proposal drafting make up the **proposal team** that will eventually sign off on the final draft to be submitted.
```

Reply:
Don't actually know if I agree on that "proposal team" comment, some folks won't draft it but might commit time and therefore should also likely sign off, for instance.

Reply:
Agreed (would rather speak of "contributors" than "drafters" etc.), but I feel good about merging and looking back on this later :)